PathOfBuildingCommunity/PathOfBuilding
# changed SimpleGraphic.dll to fix high cpu load when focused

Fixes #5292

### Description of the problem being solved:
When the PoB window gets focused the CPU Load gets to ~ 20%
When unfocused its nearly 0%

### Steps taken to verify a working solution:
- Open PoB window
- make sure its focused
- watch ressource monitor

### Link to a build that showcases this PR:
https://pobb.in/YmvFdS3O19lv

### Before screenshot:
![Screenshot 2023-04-13 195446](https://user-images.githubusercontent.com/17785769/231843840-4a35a7f1-7944-4954-914d-0ccd20c8a3ae.png)

### After screenshot:

![Screenshot 2023-04-13 203530](https://user-images.githubusercontent.com/17785769/231852083-c87f0899-0ca5-4330-abdc-6c92b2b0201a.png)


dll from:
https://github.com/Openarl/PathOfBuilding/issues/636
